### DISCUSSION TITLE
Is this library maintained?

### DISCUSSION THREAD

**Discussion by nedyalkov** [agenda/agenda#1486] [Q&A]
I've noticed this project has ~260 issues (some since 2017) without comments or anything, and ~37 PRs some addressing part of the issues, which haven't been reviewed (no single comment posted). This all seems very suspicious to me. Do you know if the library has been actively maintained, we should rather not use it for production purposes.

The reason I'm asking is, that we've started implementing it, and we need a feature. We are happy to develop it and contribute it, but if nobody is reviewing the PRs and therefore not merging them, this makes no point.

**Comment by harisvsulaiman** [comment]
@nedyalkov We are trying to maintain agenda for sometime. Although not actively adding new features. 

What was the feature you were looking for?

**Reply by vipulvyas** [reply]
Hey I want to contribute on this, i checked and using agendash for new project.

**Comment by coopbri** [comment]
@nedyalkov Agenda seems to not be actively maintained (most recent commit a few months ago as well as the issues you brought up); I know this is not ideal, but I ended up switching to [BullMQ](https://docs.bullmq.io/). Curious if you are still using Agenda or if you migrated as well.
